src/Cms/Collection.php

```diff
@@ -337,4 +339,34 @@ public function toArray(Closure $map = null): array
             return $object->toArray();
         });
     }
+
+    /**
+     * Calls a method on a Kirby object in protected mode
+     * without the permissions of the current user
+     *
+     * @param object $object
+     * @param string $attribute
+     * @return mixed
+     */
+    protected function getAttributeFromObject($object, string $attribute)
+    {
+        $auth = kirby()->auth();
+        $userBefore = $auth->currentUserFromImpersonation();
+
+        try {
+            $auth->impersonate('nobody');
+            return $object->{$attribute}();
```

Comment:
Just to make changes easier down the road: I'd actually call the parent method here. So if we make changes to that, we don't need to make them twice ideally.